txnpool/common/transaction_pool_test.go

```diff
@@ -78,11 +78,15 @@ func TestTxPool(t *testing.T) {
 		return
 	}
 
-	txList := txPool.GetTxPool(true)
+	txList, oldTxList := txPool.GetTxPool(true, 0)
 	for _, v := range txList {
 		fmt.Println(v)
 	}
 
+	for _, v := range oldTxList {
+		fmt.Println(v)
+	}
```

Comment:
use `assert` to replace all those `println` stuff

Reply:
Udpated.